managed/cmd/pmm-managed/main.go

```diff
@@ -743,6 +744,10 @@ func main() {
 	}
 	defer sqlDB.Close() //nolint:errcheck
 
+	if postgresVersion, err := ver.GetPostgreSQLVersion(ctx, sqlDB); err != nil || postgresVersion.Float() < 14 {
```

Comment:
Let's make the minimum version `14` a constant and use the constant instead of the actual number.
This will help us change the version in the future more easily.

Reply:
constant added